xlators/storage/posix/src/posix-handle.h

```diff
@@ -214,7 +214,8 @@ posix_create_link_if_gfid_exists(xlator_t *this, uuid_t gfid, char *real_path,
                                  inode_table_t *itable);
 
 int
-posix_check_internal_writes(xlator_t *this, fd_t *fd, int sysfd, dict_t *xdata);
+posix_check_internal_writes(xlator_t *this, struct _inode *fd_inode, int sysfd,
```

Comment:
for keeping sanity, can the type be 'inode_t *fd_inode' instead of 'struct _inode'. I had to check if this is something new.